ios/Podfile.lock

```diff
@@ -404,8 +402,6 @@ PODS:
     - React-Core
   - RNCAsyncStorage (1.12.1):
     - React-Core
-  - RNCPicker (1.9.11):
```

Comment:
hmm looks like this was just added and merged a few hours ago here https://github.com/Expensify/Expensify.cash/pull/1553/files

Reply:
sounds like you need to `npx pod-install` to fix? cc @marcaaron ?

Reply:
I did pod install and RNCPicker got removed. What can be the case?
Even in the master when I do pod install, It's getting removed from pod.lock

Reply:
Okay.. since the picker is newly added, I need to do run npm install. Then only pod install.

Reply:
I pulled master, ran `npm install` in the root then `cd ios/ && pod install` and looks like I'm getting the same changes so something is messed up. Asking in our Slack to try and get it figured out

<img width="631" alt="image" src="https://user-images.githubusercontent.com/4073354/109722874-88fd8d00-7b6a-11eb-88e5-35889b38e458.png">

Reply:
okay sorry for the delay. I'm pretty sure was done wrong and you have here is correct.

Looking at https://artsy.github.io/blog/2016/05/03/podspec-checksums/ and https://romainbrunie.medium.com/podfile-lock-87df8c744dc6 helped show me how to confirm the checksum. only thing to note is we install the pods into the repo directory instead of a global `~/.cocoapods`.

```
Expensidev/Expensify.cash (master) $ pod ipc spec ios/Pods/Local\ Podspecs/react-native-document-picker.podspec.json | openssl sha1
b3e78a8f7fef98b5cb069f20fc35797d55e68e28
```

looks like another conflict in the podfile.lock 😭 so if you could pull master again, `npm i && cd ios && pod install` and repush. 

I'll merge to once conflicts are gone. Thanks for bearing with me.

Reply:
@bondydaa please check now.. 😊